.gitignore

```diff
@@ -20,4 +20,4 @@
 
 npm-debug.log*
 yarn-debug.log*
-yarn-error.log*
+yarn-error.log*
```

Comment:
```suggestion
yarn-error.log*

```

Reply:
Can likely remove yarn-debug and lerna-debug as well?